Aztec/Classes/Converters/HTMLNodeToNSAttributedString.swift

```diff
@@ -240,6 +249,22 @@ class HMTLNodeToNSAttributedString: SafeConverter {
         .h3: HeaderFormatter(headerLevel: .h3),
         .h4: HeaderFormatter(headerLevel: .h4),
         .h5: HeaderFormatter(headerLevel: .h5),
-        .h6: HeaderFormatter(headerLevel: .h6)
+        .h6: HeaderFormatter(headerLevel: .h6),
+        .span: ColorFormatter()
```

Comment:
@SergioEstevao - If we want to leave it like this for this PR I'm fine with it.  I just wanted to raise awareness that we may need to change this a bit moving forward.